grakn-engine/src/main/java/ai/grakn/engine/controller/api/EntityTypeController.java

```diff
@@ -0,0 +1,182 @@
+/*
+ * Grakn - A Distributed Semantic Database
+ * Copyright (C) 2016  Grakn Labs Limited
+ *
+ * Grakn is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Grakn is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Grakn. If not, see <http://www.gnu.org/licenses/gpl.txt>.
+ */
+
+package ai.grakn.engine.controller.api;
+
+import ai.grakn.GraknTx;
+import ai.grakn.GraknTxType;
+import ai.grakn.Keyspace;
+import ai.grakn.concept.EntityType;
+import ai.grakn.concept.AttributeType;
+import ai.grakn.concept.Role;
+import ai.grakn.engine.factory.EngineGraknTxFactory;
+import mjson.Json;
+import org.apache.commons.httpclient.HttpStatus;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.Request;
+import spark.Response;
+import spark.Service;
+
+import java.util.Optional;
+
+import static ai.grakn.engine.controller.util.Requests.mandatoryBody;
+import static ai.grakn.engine.controller.util.Requests.mandatoryPathParameter;
+import static ai.grakn.engine.controller.util.Requests.mandatoryQueryParameter;
+import static ai.grakn.util.REST.Request.ENTITY_TYPE_LABEL_PARAMETER;
+import static ai.grakn.util.REST.Request.KEYSPACE;
+import static ai.grakn.util.REST.WebPath.Api.ENTITY_TYPE;
+import static ai.grakn.util.REST.WebPath.Api.ENTITY_TYPE_ATTRIBUTE_TYPE_ASSIGNMENT;
+import static ai.grakn.util.REST.WebPath.Api.ENTITY_TYPE_ROLE_ASSIGNMENT;
+
+/**
+ * <p>
+ *     EntityType endpoints
```

Comment:
`{@link EntityType} endpoints`

Reply:
fixed in 4bf97bbec55860dda114844663b22a5f412587ec